rplugin/python3/denite/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 if not find_loader('vim'):
     import neovim
-    from denite.ui.default import Default
+    from denite.ui.bridge import Bridge
```

Comment:
I think this might be changed in future release because denite will have custom UI support. That said, should neovim-prompt be a separate package rather built into denite core?